File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/bean/KNNDistance.java
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean;

public class KNNDistance {

    // 已知点id
    private long id;
    // 未知点id
    private long nid;
    // 二者之间的距离
    private double disatance;

    public KNNDistance() {
    }

    public KNNDistance(long id, long nid, double disatance) {
        this.id = id;
        this.nid = nid;
        this.disatance = disatance;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getNid() {
        return nid;
    }

    public void setNid(long nid) {
        this.nid = nid;
    }

    public double getDisatance() {
        return disatance;
    }

    public void setDisatance(double disatance) {
        this.disatance = disatance;
    }
}

File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/bean/KNNDistance.kt
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean

class KNNDistance {

    // 已知点id
    var id: Long = 0
    // 未知点id
    var nid: Long = 0
    // 二者之间的距离
    var disatance: Double = 0.toDouble()

    constructor() {}

    constructor(id: Long, nid: Long, disatance: Double) {
        this.id = id
        this.nid = nid
        this.disatance = disatance
    }
}
